tests/describe_task_queue.go

```diff
@@ -39,27 +39,70 @@ import (
 )
 
 type (
-	DescribeTaskQueueSuite struct {
+	DescribeTaskQueueSuiteBase struct {
 		*require.Assertions
 		FunctionalTestBase
 	}
+
+	// DescribeTaskQueueSuite tests DescribeTaskQueue calls with the smaller cache TTL than one second.
+	// This is done to test if DescribeTaskQueue calls return the most updated information from the server with large
+	// cache evictions
+	DescribeTaskQueueSuite struct {
+		DescribeTaskQueueSuiteBase
+	}
+
+	// DescribeTaskQueueSuiteWithCache tests DescribeTaskQueue calls with a larger cache TTL than one second.
+	// This is done to test if the cached responses are as expected.
+	DescribeTaskQueueSuiteWithCache struct {
+		DescribeTaskQueueSuiteBase
+	}
 )
 
-func (s *DescribeTaskQueueSuite) SetupSuite() {
+var smallTTL = time.Nanosecond
+var largeTTL = 3 * time.Second
+
+func (s *DescribeTaskQueueSuiteBase) SetupSuite() {
 	s.setupSuite("testdata/es_cluster.yaml")
 }
 
-func (s *DescribeTaskQueueSuite) TearDownSuite() {
+func (s *DescribeTaskQueueSuiteBase) TearDownSuite() {
 	s.tearDownSuite()
 }
 
-func (s *DescribeTaskQueueSuite) SetupTest() {
+func (s *DescribeTaskQueueSuiteBase) SetupTest() {
 	s.FunctionalTestBase.SetupTest()
-
 	// Have to define our overridden assertions in the test setup. If we did it earlier, s.T() will return nil
 	s.Assertions = require.New(s.T())
 }
 
+func (s *DescribeTaskQueueSuite) SetupSuite() {
+	s.dynamicConfigOverrides = map[dynamicconfig.Key]any{
+		dynamicconfig.TaskQueueInternalInfoCacheTTL.Key(): smallTTL,
+	}
+	s.DescribeTaskQueueSuiteBase.SetupSuite()
+}
+func (s *DescribeTaskQueueSuite) TearDownSuite() {
```

Comment:
do you need to define this method if you're not overriding the base implementation?